src/index.js

```diff
@@ -0,0 +1,92 @@
+const crypto = require('crypto');
+
+function string2Boolean(str) {
+  return (typeof str === 'boolean' ? `${str}` : str);
+}
+
+function boolean2String(bool) {
+  return (['true', 'false'].includes(bool) ? bool === 'true' : bool);
+}
+
+const bytesToGenerate = 12;
+const vector = crypto.randomBytes(bytesToGenerate);
```

Comment:
Must not be module-scoped
[Reference](https://gist.github.com/chris-rock/fe87dd35d6168512a2f7#file-crypto-gcm-js-L8)